projects/arlas-toolkit/src/lib/components/aias-download/aias-download.component.html

```diff
@@ -66,6 +68,12 @@
             ></span>
             <span class="infos-subtitle">{{ 'Summary of parameters' | translate }} :</span>
           </div>
+
+          <div class="params">
+            <span class="label">{{ 'Download type'| translate }} :</span>
+            <span class="value">{{ (formGroup.get('raw_archive').value) ? 'Archive' : 'Image' | translate }}</span>
```

Comment:
If we want the keys to be properly extracted, I think you need to put the translate pipe right after the key, but you should test if that is the case first